source/2019-12-17-countdown-to-the-new-year-2019-ember-leaflet.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-leaflet
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber
+alias: 2019/12/17/countdown-to-the-new-year-2019-ember-leaflet
+responsive: true
+---
+
+The first of its kind "Countdown to The New Year: 31 Days of Ember Addons" series highlights popular addons in the Ember community. We hope you enjoy learning about fun tools for Ember developers together with us each day!
+
```

Comment:
add `READMORE`

Reply:
* I guess the proper place based on the other posts pattern is before `What It Does`

Reply:
Yes, good call - I can update this accordingly :+1: